test/js/web/fetch/fetch-gzip.test.ts

```diff
@@ -18,7 +18,7 @@ it("fetch() with a buffered gzip response works (one chunk)", async () => {
   });
   gcTick(true);
 
-  const res = await fetch(`http://${server.hostname}:${server.port}`, { verbose: true });
+  const res = await fetch(server.url, { verbose: true });
```

Comment:
There’s no way this makes the test more reliable 

Easier to write though

Reply:
That test will fail if the `hostname` is an IPv6 address

Reply:
sometimes the server get a IPV6 and the concat will be a invalid url because will need `[::1]` not only `::1`

Reply:
That's one of the big reasons for `server.url` to exist. We should audit all usages of `server.hostname` + `server.port` in our tests and convert them to `server.url`

Reply:
> That's one of the big reasons for `server.url` to exist. We should audit all usages of `server.hostname` + `server.port` in our tests and convert them to `server.url`

Im doing that